unit/solvers/lowering/byte_operators.cpp

```diff
@@ -154,8 +154,7 @@ SCENARIO("byte_extract_lowering", "[core][solvers][lowering][byte_extract]")
             const exprt lower_be = lower_byte_extract(be, ns);
             const exprt lower_be_s = simplify_expr(lower_be, ns);
 
-            // TODO: does not currently hold
-            // REQUIRE(!has_subexpr(lower_be, ID_byte_extract_little_endian));
+            REQUIRE(!has_subexpr(lower_be, ID_byte_extract_little_endian));
```

Comment:
Presumably also big_endian?

Reply:
Ah, indeed, actually that should be fixed in the first commit (which is #4061).

Reply:
Done.